test/support/vcr.rb

```diff
@@ -2,6 +2,8 @@
 require "vcr"
 require "active_support/concern"
 
+WebMock.disable_net_connect!(:allow_localhost => true)
```

Comment:
this isn't part of the specs. do we want to include this? because it will affect all tests.